Library/Homebrew/cmd/search.rb

```diff
@@ -102,8 +102,9 @@ def query_regexp(query)
 
   def search_taps(query)
     valid_dirnames = ["Formula", "HomebrewFormula", "Casks", "."].freeze
-    matches = GitHub.search_code("user:Homebrew", "user:caskroom", "filename:#{query}", "extension:rb")
-    [*matches].map do |match|
+    matches = GitHub.search_code(user: ["Homebrew", "caskroom"], filename: query, extension: "rb")
```

Comment:
That'd be kinda difficult to pull off, since `#query_string` would have to know how to singularize a word. I could just make it remove an `s` at the end of the word, or hard-code the `users` case, but that would make the method too specific in my opinion